usage.go

```diff
@@ -215,10 +233,14 @@ func (p *Parser) writeHelpForCommand(w io.Writer, cmd *command) {
 }
 
 func (p *Parser) printOption(w io.Writer, spec *spec) {
-	left := synopsis(spec, "--"+spec.long)
+	text := make([]string, 0, 2)
+	if spec.long != "" {
+		text = append(text, synopsis(spec, "--"+spec.long))
+	}
 	if spec.short != "" {
-		left += ", " + synopsis(spec, "-"+spec.short)
+		text = append(text, synopsis(spec, "-"+spec.short))
 	}
+	left := strings.Join(text, ", ")
 	printTwoCols(w, left, spec.help, spec.defaultVal, spec.env)
```

Comment:
would you put a guard  `if left != ""` around this call and remove the guard from inside printTwoCols?